networktest.go

```diff
@@ -252,16 +254,16 @@ func (l *lightningNode) stop() error {
 	// - start of the node wasn't initiated
 	// - process wasn't spawned
 	// - process already finished
-	processFinished := l.cmd.ProcessState != nil &&
-		l.cmd.ProcessState.Exited()
-	if l.cmd == nil || l.cmd.Process == nil || processFinished {
-		return nil
-	}
 
-	if runtime.GOOS == "windows" {
-		return l.cmd.Process.Signal(os.Kill)
+	select {
```

Comment:
Hmm, I'm not quit sure I follow the rational here. The race was caused the access to `l.cmd` above concurrently with the goroutine exiting above when it reads from the variables internal to the `Wait` method.